compiler/rustc_target/src/spec/x86_64_unknown_none_elf.rs

```diff
@@ -0,0 +1,31 @@
+// Generic AArch64 target for bare-metal code - Floating point disabled
+//
+// Can be used in conjunction with the `target-feature` and
+// `target-cpu` compiler flags to opt-in more hardware-specific
+// features.
+//
+// For example, `-C target-cpu=cortex-a53`.
+
+use super::{CodeModel, LinkerFlavor, LldFlavor, PanicStrategy, Target, TargetOptions};
+
+pub fn target() -> Target {
+    let opts = TargetOptions {
+        abi: "softfloat".to_string(),
```

Comment:
This shouldn't be here. `softfloat` is an ARM ABI, not an x86 one. The target just shouldn't have an `ABI` field.